### PR TITLE
setup basic folders

### PR DESCRIPTION
**Pull request by kevinrczhang**
- postgresql db setup is very difficult, you'll need to fun pgadmin4 while running the app to be able to work with it, so for local, best to use sqlite
- we can still use postgresql for prod, will just have to change the database specs, but that can be handled later

# Description

Please include a summary of the change and which issue is fixed. Add any screenshots of the change.

Fixes # (ticket #)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: backend/auction/models.py
Comment (ltan02) [inline backend/auction/models.py:1]: @kevinrczhang Would it be better to split this up into separate apps, like:
- Auction
- Truck and Brand
- Bid

I think this approach will be beneficial in the future if we need to update our models and different people are working on those tasks.

Comment (kevinrczhang) [reply backend/auction/models.py:1]: The bid itself is not complex enough to have its own app IMO, but I do agree with truck and brand being in its separate app. I'll do that, which means we'd have 3 apps in total

File: backend/product/models.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from core.models import MainModel
+
+class Brand(MainModel):
+    name = models.CharField(max_length=150, null=False)
+
+class Truck(MainModel):
```
Comment (ltan02) [inline backend/product/models.py:7]: There are a couple more fields that we will need:
- Unicode ID
- Chassis Number
- Model Number
- Description

File: backend/product/models.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from core.models import MainModel
+
```
Comment (ltan02) [inline backend/product/models.py:3]: Another model that we will also need is `VehicleType` with `name` attribute as well, which the `Vehicle` model will have a foreign key to

File: backend/product/models.py (hunk 1 above)
Comment (ltan02) [inline backend/product/models.py:7]: Can we rename this to `Vehicle`

**Comment by ltan02** [comment]
Instead of naming the app `Product`, can we rename it to `Vehicle`

File: backend/vehicle/models.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    
+
+class Vehicle(MainModel):
+    date = models.DateTimeField(editable=False)
+    asking_price = models.IntegerField(null=False)
+    brand = models.ForeignKey(Brand, on_delete=models.PROTECT)
+    description = models.CharField(max_length=2000)
+    auction = models.ForeignKey("auction.Auction", on_delete=models.PROTECT)
+    unicode_id = models.IntegerField()
+    model_number = models.IntegerField()
```
Comment (ltan02) [inline backend/vehicle/models.py:15]: Missing foreign key to VehicleType

**Review by ltan02 — APPROVED** [review]

**Pull request merged**